changelog.md

```diff
@@ -6,12 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+- [[#3463] API Docs Updated](https://github.com/ballerina-platform/ballerina-standard-library/issues/3463)
```

Comment:
```suggestion
- [API Docs Updated](https://github.com/ballerina-platform/ballerina-standard-library/issues/3463)
```

Reply:
> 
We are following the same pattern in pervious entries. similar pattern followed in graphql changelog as well, Shall we keep this?